bridges/TwitterBridge.php

```diff
@@ -234,7 +234,31 @@ public function collectData(){
 
 			// generate the title
 			$item['title'] = $tweet->full_text;
-			$cleanedTweet  = $tweet->full_text;
+
+			// Convert plain text URLs into HTML hyperlinks
+			if (isset($tweet->entities->media)) {
+				foreach($tweet->entities->media as $media) {
+					$cleanedTweet = str_replace($media->url,
+						'<a href="' . $media->expanded_url . '">' . $media->display_url . '</a>',
+						$tweet->full_text);
```

Comment:
If tweet has more than one media - only last media link will be converted. Perhaps you should return back removed `$cleanedTweet  = $tweet->full_text;` upper and use $cleanedTweet as third param in str_replace.

Reply:
my bad and fixed.